content/sensu-core/2.0/guides/create-read-only-user.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "How to create a Read Only User with RBAC"
+linkTitle: "Create Read Only User"
+weight: 1
+version: "2.0"
+product: "Sensu Core"
+platformContent: False
+menu: 
+  sensu-core-2.0:
+    parent: guides
+---
+
+## What is RBAC?
+**Role-Based Access Control** (RBAC) is Sensu's local user management system. RBAC currently supports the management of users and permissions with Organizations, Environments, Roles, and Users. Support for external directories (such as LDAP) will be available in a future release.
```

Comment:
As per [Matt suggestion](https://github.com/sensu/sensu-docs-site/pull/223#discussion_r171637928), I think we should decapitalize `Organizations, Environments, Roles, and Users` and bold these terms instead. What do you think?

Reply:
That's fine with me! I'll update it.